src/job/models/ingestion/seedingJobCreator.ts

```diff
@@ -40,29 +41,31 @@ export class SeedingJobCreator {
           ingestionJobId: ingestionJob.id,
           seedJobType: this.seedJobType,
           seedTaskType,
-          mode,
           layerName,
         });
 
         logger.debug({ msg: 'Getting cache name for layer', layerName });
+
         const cacheName = await this.mapproxyClient.getCacheName({ layerName, cacheType: LayerCacheType.REDIS });
         activeSpan?.addEvent('getCacheName.success', { cacheName });
 
-        const geometry = this.calculateGeometryByMode(mode, ingestionJob);
+        const validCatalogId = internalIdSchema.parse(ingestionJob).internalId;
 
-        if (!geometry) {
-          activeSpan?.addEvent('calculateGeometry.empty');
-          logger.warn({ msg: 'No geometry found, skipping seeding job creation' });
-          return;
-        }
-        activeSpan?.addEvent('calculateGeometry.success', { geometry: JSON.stringify(geometry) });
+        const seedTasks: {
```

Comment:
modify `SeedTaskParams` to include `type: string` then it will become
`seedTasks: SeedTaskParams[] = []`

Reply:
SeedTaskParams describe only the task parameters

Reply:
"type" is actually the type of the task itself so is cannot be contained in the "partameters" of the seed tasks, but you should use here:
const seedTasks: ICreateTaskBody<SeedTaskParams>[] = []
to define the task body request of an seed tasks